### DISCUSSION TITLE
Export recent connections list to file for importing in different version

### DISCUSSION THREAD

**Discussion by cody151** [rustdesk/rustdesk#8793] [General]
I am having issues with the most recent version 1.2.6 so I would like to downgrade to a lower version 1.2.3 where everything was working fine. However I need to uninstall before installing the old version but I need to retain my recent connections list.

**Comment by andy-page-developer** [comment]
Would love to see this. I have 20 computers on my LAN and setting up new computers is a bit of a pain. Would be great to an easy Export/Import funtion
